docs/data/data-grid/filtering/index.md

```diff
@@ -103,6 +103,26 @@ You can use the `onFilterModelChange` prop to listen to changes to the filters a
 
 {{"demo": "ControlledFilters.js", "bg": "inline", "defaultCodeOpen": false}}
 
+### Read-only filters
+
+You can initialize the `filterModel`, set the `filterModel` prop, or use `apiRef.current.setFilterModel` to define the filters for columns with `colDef.filterable` set to `false`. These filters will be applied but the user won't be able to change them.
```

Comment:
A small possible positive side effect would be [this kind of filtering](https://github.com/mui/mui-x/issues/8357) will now be [possible](https://codesandbox.io/p/sandbox/stoic-brattain-dpyvjz?file=%2Fsrc%2FDemo.tsx%3A129%2C56) in read-only mode without [tweaking](https://github.com/mui/mui-x/issues/8357#issuecomment-1483231110) `groupingColDef.`